2-multitenant/envs/development/outputs.tf

```
@@ -54,6 +54,11 @@ output "app_certificates" {
   value       = module.env.app_certificates
 }
 
+output "acronym" {
```

As this is a list, should it be plural?  Just an idea, would it be helpful to just pass through all of `var.apps`?

```suggestion
output "acronyms" {
```